scripts/make-bundle

```diff
@@ -155,13 +156,14 @@ def verify_preconditions():
 def _min_version_required(min_version, actual_version, name):
     # precondition: min_version is major.minor.patch
     #               actual_version is major.minor.patch
-    min_split = min_version.split('.')
-    actual_split = actual_version.split('.')
-    for minimum, actual in zip(min_split, actual_split):
-        if int(actual) < int(minimum):
-            raise ValueError("%s requires at least version %s, but "
-                             "version %s was found." % (name, min_version,
-                                                        actual_version))
+    min_split = map(int, min_version.split('.'))
+    actual_split = map(int, actual_version.split('.'))
+    comp = map(cmp, actual_split, min_split)
```

Comment:
We don't need the cmp and reduce calls here.  The default __gt__ for lists does what we want:

```
>>> [1,7,8] < [1,7,9]
True
>>> [1,7,8] < [1,7,7]
False
>>> [1,10,8] < [1,9,7]
False
>>> [1,10,8] < [1,10,10]
True
```